gazelle/config.go

```diff
@@ -147,11 +149,13 @@ func (*swiftLang) Configure(c *config.Config, rel string, f *rule.File) {
 		switch d.Key {
 		case moduleNamingConventionDirective:
 			if d.Value == swiftcfg.PascalCaseModuleNamingConvention {
+				log.Println("module naming convention ", swiftcfg.PascalCaseModuleNamingConvention)
```

Comment:
Is this necessary? In the future, it would be nice to have a verbose mode that told us this kind of information. I am unsure that we should just print it out by default.

Reply:
No sorry I was debugging something and forgot to take these out. I'll remove them.